documentation/src/docs/clients-and-streams.md

```diff
@@ -252,49 +397,83 @@ Often, however, you’ll need to preserve the order of related events, and this
 
-Map<String, Integer> decades = new LinkedHashMap<>();
-decades.put("ones", 0);
-decades.put("tens", 10);
-decades.put("twenties", 20);
-decades.entrySet().stream().forEach(decade -> {
-    IntStream.range(decade.getValue(), decade.getValue() + 10)
-    .forEachOrdered(n -> {
-        try {
-            decWriter.writeEvent(decade.getKey(), n);
-        } catch (InterruptedException | ExecutionException e) {
-            throw new RuntimeException(e);
-        }
+    Map<String, Integer> decades = new LinkedHashMap<>();
+    decades.put("ones", 0);
+    decades.put("tens", 10);
+    decades.put("twenties", 20);
+    decades.entrySet().stream().forEach(decade -> {
+        IntStream.range(decade.getValue(), decade.getValue() + 10)
+        .forEachOrdered(n -> {
+            try {
+                decWriter.writeEvent(decade.getKey(), n);
+            } catch (InterruptedException | ExecutionException e) {
+                throw new RuntimeException(e);
+            }
+        });
     });
-});
-decWriter.close();
-```
+    decWriter.close();
+    ```
+
+=== "Python"
+
+    ``` python
+    import pravega_client
+
+    manager=pravega_client.StreamManager("tcp://127.0.0.1:9090")
+
+    # assuming the Pravega scope and stream are already created.
+    paraWriter=manager.create_writer("tutorial", "parallel-decades")
+    
+    for x in range(10):
+        paraWriter.writeEvent(x, "ones")
+    for y in range(20):
+        paraWriter.writeEvent(y, "tens")
+    for z in range(20):
+        paraWriter.writeEvent(z, "twenties")
+    ```
 
 The “ones” (0-9), “tens” (10-19) and “twenties” (20-29) are written to our `parallel-decades` stream. Since we have 3 sequences to 5 segments, each decade sequence is potentially in its own segment.
 
 When we read this back, the values of each decade will be in order, but the sequences may be interleaved, and the lower decades won’t necessarily come first:
 
-```java
-ReaderGroupConfig readerGroupConfig = ReaderGroupConfig.builder()
-        .stream("tutorial/parallel-decades").build();
-readerGroupManager.createReaderGroup("paraDecReader", readerGroupConfig);
-EventStreamReader<Integer> reader = factory
-        .createReader("decId", "paraDecReader",
-                      new JavaSerializer<Integer>(), ReaderConfig.builder().build());
-Integer intEvent;
-while ((intEvent = reader.readNextEvent(1000).getEvent()) != null) {
-    System.out.println(intEvent);
-}
-reader.close();
-```
+=== "Java"
 
+    ``` java
+    ReaderGroupConfig readerGroupConfig = ReaderGroupConfig.builder()
+            .stream("tutorial/parallel-decades").build();
+    readerGroupManager.createReaderGroup("paraDecReader", readerGroupConfig);
+    EventStreamReader<Integer> reader = factory
+            .createReader("decId", "paraDecReader",
+                      new JavaSerializer<Integer>(), ReaderConfig.builder().build());
+    Integer intEvent;
+    while ((intEvent = reader.readNextEvent(1000).getEvent()) != null) {
```

Comment:
same here

Reply:
To skip checkpoint event, i have updated while condition.